INSTALL.rst

```diff
@@ -32,7 +32,7 @@ Make a conda environment with the latest packages for Python 3.8 or
 higher with the name *mdpow*; this installs the larger dependencies that are
 pre-requisites for MDPOW::
 
-  conda create -c conda-forge -n mdpow  numpy scipy matplotlib mdanalysis pyyaml alchemlyb pandas gromacswrapper
+  conda create -c conda-forge -n mdpow  numpy scipy matplotlib mdanalysis pyyaml alchemlyb pandas gromacswrapper rdkit
```

Comment:
add seaborn, which is already included in `MDPOW/doc/requirements.txt`

Reply:
thanks, addressed in 301ef52c2b55be7ed78acd61b196a8dcf5af3eea